templates/partials/categories/lastpost.tpl

```diff
@@ -15,7 +15,7 @@
 
 	<!-- IF !posts.length -->
 	<div class="post-content">
-		No new posts.
+		<div class="no_posts">[[category:no_posts:No new posts.]]</div>
```

Comment:
`<div class="no_posts">[[category:no_posts]]</div>`. The english translation should be saved in `public/language/en_GB/category.json` under the key `no_posts`.